project_euler/problem_050/__init__.py

```diff
@@ -0,0 +1 @@
+

```

Comment:
Try making a completly empty `__init__.py` file with no content and not even a single new line